src/multi.c

```diff
@@ -342,6 +347,20 @@ void unwatchAllKeys(client *c) {
     }
 }
 
+/* iterates over the watched_keys list and
+ * look for an expired key . */
+bool isWatchedKeyExpiredExists(client *c) {
+    listIter li;
+    listNode *ln;
+    watchedKey *wk;
+    if (listLength(c->watched_keys) == 0) return false;
+    listRewind(c->watched_keys,&li);
+    while((ln = listNext(&li))) {
+        wk = listNodeValue(ln);
+        if (keyIsExpired(c->db, wk->key)) return true;
+    }
```

Comment:
Missing a ```return false``` here.

Reply:
note, `0`, not `false` (same goes for the `true` above)